packages/usa-in-page-navigation/src/test/test-patterns/test-custom-header-selector.twig

```diff
@@ -0,0 +1,32 @@
+<div class="usa-in-page-nav-container">
+  <aside class="usa-in-page-nav"
+    {% if includeHeaders == "All" %}
+      data-heading-selector="h1 h2 h3 h4 h5 h6"
```

Comment:
It does only accept html header elements. I think it could be a good future enhancement to also allow for an option to exclude or include headers with designated selectors, but that is not a part of this work. 

For clarity, I've updated the name to `data-heading-elements`.

Reply:
New attribute looks good and found no references to the old one.